File: nova-cli/src/main/java/org/nectar/nova/cli/command/ServeCommand.java
package org.nectar.nova.cli.command;

import io.javalin.Javalin;
import lombok.val;
import picocli.CommandLine;

@CommandLine.Command
public class ServeCommand implements Runnable {

    @Override
    public void run() {
        val lin = Javalin.create(javalinConfig -> {
        });
        lin.start();
    }
}

File: nova-cli/src/main/kotlin/org/nova/cli/command/ServeCommand.kt
package org.nova.cli.command

import picocli.CommandLine.Command
import java.util.concurrent.Callable

@Command(name = "server", aliases = ["s"], mixinStandardHelpOptions = true)
class ServeCommand :Callable<Int>{
    override fun call(): Int {

        return 0;
    }
}